app/assets/stylesheets/main.scss

```diff
@@ -0,0 +1,247 @@
+    
+    accent-color: #000000;
+    margin: 10px 220px 10px 10px;
+
+}
+
+
+.btn{
+    
+    background: #fff;
+    border-radius: 40px;
+    border: none;
+    color: #333;
+    cursor: pointer;
+    font-size: 16px;
+    font-weight: 600;
+    height: 40px;
+    outline: none;
+    align-items: center;
+    width: 80%;
+    margin: 0 10px 20px 30px
+
+}
+
+
+
+.wrapper .register-link{
+    font-size: 14.5px;
+    margin: 20px 0 15px;
+    text-align: center;
+}
+
+.register-link p a {
+    color: #000000;
+    font-weight: 600;
+    text-decoration: none;
+}
+
+.register-link p a:hover {
+    text-decoration: underline;
+}
+
+.sig{
+    margin: 0 40px 0 40px;
+
+}
+
+
+
+@include desktop-lg {
+    .logo{
+        display: grid;
+        
+        place-items: center;
+        
+        h1 {
+            font-family: 'Black Ops One', sans-serif;
+            font-weight: 100;
+            
+        }
+        
+    }
+    
+    body {
+        align-items: center;
+        background: #FEFCE8;
+        display: flex;
+        flex-direction: column;
+        gap: 1rem;
+        justify-content: center;
+        min-height: 40vh;
+    }
+    
+    .wrapper {
+        
+        background: #D9D9D9;
+        border-radius: 10px;
+        color: #000000;
+        padding: 20px 10px ;
+    }
+    
+    .wrapper h2{
+        font-size: 36px;
+        text-align: center;
+    } 
+    
+    .wrapper .input-box{
+        height: 45px;
+        margin: 25px 0;
+        position: relative;
+        width: 100%;
+    }
+    
+    .input-box input {
+        border-radius: 40px;
+        border: 2px solid rgba(255, 255, 255, .2);
+        border: none;
+        color: #0a0a0a;
+        font-size: 16px;
+        height: 50%;
+        outline: none;
+        padding: 20px 45px 20px 20px;
+        width: 100%;
+    }
+    
+    .input-box input::placeholder{
+        color: #000000;
+    }
+    
+    
+    .wrapper .remember-forgot{
+        display: flex;
+        font-size: 14.5px;
+        justify-content: space-between;
+        margin: -8px 0 8px;
+    }
+    
+    .remember-forgot label, check_box{
+        
+        accent-color: #000000;
+        margin: 10px 220px 10px 10px;
+    
+    }
+    
+    
+    .btn{
+        
+        background: #fff;
+        border-radius: 40px;
+        border: none;
+        color: #333;
+        cursor: pointer;
+        font-size: 16px;
+        font-weight: 600;
+        height: 40px;
+        outline: none;
+        align-items: center;
+        width: 80%;
+        margin: 0 10px 20px 30px
+    
+    }
+    
+    
+    
+    .wrapper .register-link{
+        font-size: 14.5px;
+        margin: 20px 0 15px;
+        text-align: center;
+    }
+    
+    .register-link p a {
+        color: #000000;
+        font-weight: 600;
+        text-decoration: none;
+    }
+    
+    .register-link p a:hover {
+        text-decoration: underline;
+    }
+    
+    .sig{
+        margin: 0 40px 0 40px;
+    
+    }
+}
+

```

Comment:
I recommend using something like sass 7.1 pattern to order files since as your application grows it will get harder and harder to mantain it